client/lib/plans/features-list.js

```diff
@@ -708,44 +708,44 @@ export const FEATURES_LIST = {
 	},
 	[ constants.FEATURE_BACKUP_ARCHIVE_30 ]: {
 		getSlug: () => constants.FEATURE_BACKUP_ARCHIVE_30,
-		getTitle: () => i18n.translate( '30-day Backup Archive' ),
+		getTitle: () => i18n.translate( '30-day backup archive' ),
 		getDescription: () =>
 			i18n.translate( 'Browse or restore any backup made within the past 30 days.' ),
 	},
 	[ constants.FEATURE_BACKUP_ARCHIVE_15 ]: {
 		getSlug: () => constants.FEATURE_BACKUP_ARCHIVE_15,
-		getTitle: () => i18n.translate( '15-day Backup Archive' ),
+		getTitle: () => i18n.translate( '15-day backup archive' ),
 		getDescription: () =>
 			i18n.translate( 'Browse or restore any backup made within the past 15 days.' ),
 	},
 	[ constants.FEATURE_BACKUP_ARCHIVE_UNLIMITED ]: {
 		getSlug: () => constants.FEATURE_BACKUP_ARCHIVE_UNLIMITED,
-		getTitle: () => i18n.translate( 'Unlimited Backup Archive' ),
+		getTitle: () => i18n.translate( 'Unlimited backup archive' ),
 		getDescription: () =>
 			i18n.translate( 'Browse or restore any backup made since you activated the service.' ),
 	},
 	[ constants.FEATURE_BACKUP_STORAGE_SPACE_UNLIMITED ]: {
 		getSlug: () => constants.FEATURE_BACKUP_STORAGE_SPACE_UNLIMITED,
-		getTitle: () => i18n.translate( 'Unlimited Backup Storage Space' ),
+		getTitle: () => i18n.translate( 'Unlimited backup storage space' ),
 		getDescription: () =>
 			i18n.translate( 'Absolutely no limits on storage space for your backups.' ),
 	},
 	[ constants.FEATURE_AUTOMATED_RESTORES ]: {
 		getSlug: () => constants.FEATURE_AUTOMATED_RESTORES,
-		getTitle: () => i18n.translate( 'Automated Restores' ),
+		getTitle: () => i18n.translate( 'Automated restores' ),
 		getDescription: () =>
 			i18n.translate( 'Restore your site from any available backup with a single click.' ),
 	},
 	[ constants.FEATURE_EASY_SITE_MIGRATION ]: {
 		getSlug: () => constants.FEATURE_EASY_SITE_MIGRATION,
-		getTitle: () => i18n.translate( 'Easy Site Migration' ),
+		getTitle: () => i18n.translate( 'Easy site migration' ),
 		getDescription: () =>
 			i18n.translate( 'Easily and quickly move or duplicate your site to any location.' ),
 	},
 	[ constants.FEATURE_MALWARE_SCANNING_DAILY ]: {
 		getSlug: () => constants.FEATURE_MALWARE_SCANNING_DAILY,
 		getTitle: () =>
-			i18n.translate( '{{strong}}Daily{{/strong}} Malware Scanning', {
+			i18n.translate( '{{strong}}Daily{{/strong}} malware scanning', {
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.